src/.clang-tidy

```diff
@@ -0,0 +1,28 @@
+---
+Checks:          '-*,
+                 # modernize-use-auto,
+                 # modernize-use-equals-delete,
+                 # modernize-deprecated-headers alreday applied, but stuck with https://github.com/nodejs/node/pull/26159#issuecomment-464417206
+                 modernize-make-unique,
+                 modernize-make-shared,
+                 modernize-redundant-void-arg,
+                 modernize-replace-random-shuffle,
+                 modernize-shrink-to-fit,
+                 modernize-use-bool-literals,
+                 modernize-use-emplace,
+                 modernize-use-equals-default,
+                 modernize-use-nullptr,
+                 modernize-use-override,
+                 performance-faster-string-find,
+                 readability-delete-null-pointer,
+                 performance-unnecessary-value-param
+WarningsAsErrors: ''
+HeaderFilterRegex: ''
+AnalyzeTemporaryDtors: false
+FormatStyle:     none
+User:            nodejs/cpp
+CheckOptions:
+  - key:             google-readability-braces-around-statements.ShortStatementLines
```

Comment:
I add this options to make showcase how options are put like https://github.com/nodejs/node/pull/26840/files#r267804403.